### ISSUE TITLE
Add code suggestions to comments

### ISSUE THREAD

**Issue by elbrujohalcon** [inaka/elvis_core#167]
When possible, add code suggestions to the warnings.
As an example,
Instead of…

> - macro_names
>   - Invalid macro name the_macro on line 11. Use UPPER_CASE.

…say…

> - macro_names
>   - Invalid macro name the_macro on line 11. Use UPPER_CASE.
>   - QuickFix: replace it by `THE_MACRO`

**Comment by paulo-ferraz-oliveira** [comment]
I started to look at this, but not all, I guess, require a "quick fix", especially since it might not be so quick. How do you propose a fix for `line_length`, for example? "Use a formatter?" 😄 I'll propose a few (when I have some time to code) and maybe we can go from there.

**Comment by elbrujohalcon** [comment]
Yeah… I agree that some rules don't lend themselves easily to _Quick Fixes_. So, maybe… we can say "There is no quick fix here" for those.
